data_validation/result_handlers/postgres.py

```diff
@@ -33,6 +34,42 @@
     from ibis.backends.base import BaseBackend
 
 
+def psql_insert_copy(table, conn, keys, data_iter):
```

Comment:
```suggestion
def _psql_insert_copy(table, conn, keys, data_iter):
```
Shouldn't this method be private? As it's used only internally by the actual external methods.

Reply:
Yes, good point.